link/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.23.3-alpine AS builder
+
+RUN apk update && apk add --no-cache git wget bash
+WORKDIR /app
+ENV GOPROXY=direct
+
+COPY ../../go.mod ../../go.sum ./
+RUN go mod download
+COPY ../../. .
```

Comment:
sometimes it is dangerous to copy everything, better to be more specific - to copy only thing you need